scripts/vm/network/security_group.py

```diff
@@ -607,12 +622,16 @@ def default_network_rules(vm_name, vm_id, vm_ip, vm_ip6, vm_mac, vif, brname, se
         if vm_ip:
             execute("iptables -A " + vmchain_default + " -m physdev --physdev-is-bridged --physdev-in " + vif + " -m set ! --match-set " + vmipsetName + " src -j DROP")
             execute("iptables -A " + vmchain_default + " -m physdev --physdev-is-bridged --physdev-out " + vif + " -m set ! --match-set " + vmipsetName + " dst -j DROP")
-            execute("iptables -A " + vmchain_default + " -m physdev --physdev-is-bridged --physdev-in " + vif + " -m set --match-set " + vmipsetName + " src -p udp --dport 53  -j RETURN ")
```

Comment:
@phsm 
I think we should not change it

RETURN means checks on other chains will be performed. But they will not be checked if this is changed to ACCEPT

Reply:
btw: I did not look into the changes. Correct me if I am wrong.

This script is very important for public cloud providers, we have to be very careful

Reply:
@phsm 
thanks for the reply.
as far as I remember, both ingress rules and egress rules should be checked. this is why I questioned.